ui/admin/app/components/worker-diagram/dual-filter/hcp/index.hbs

```diff
@@ -1,102 +1,91 @@
-          {{t 'resources.target.workers.diagram.egress-worker'}}
-        </D.Item>
-        <D.Item @icon='server'>
-          {{t 'resources.target.workers.diagram.host'}}
-        </D.Item>
-      </D.Group>
-    </OrderedSeriesDiagram>
-    <div class='target-worker-filter-explainer'>
-      <Hds::Form::HelperText>
-        {{t
-          'resources.target.workers.filter-explainer.dual-egress-on-ingress-on'
-        }}
-      </Hds::Form::HelperText>
-    </div>
-  {{else if @ingressFilter}}
-    <OrderedSeriesDiagram
-      data-test-dual-filter-hcp-egress-off-ingress-on
-      as |D|
+{{#if (and @egressWorkerFilterEnabled @ingressWorkerFilterEnabled)}}
+  <OrderedSeriesDiagram data-test-dual-filter-hcp-egress-on-ingress-on as |D|>
+    <D.Item @icon='user'>
+      {{t 'resources.target.workers.diagram.client'}}
+    </D.Item>
+    <D.Item @icon='settings'>
+      {{t 'resources.target.workers.diagram.ingress-worker'}}
+    </D.Item>
+    <D.Group
+      @title={{t 'resources.target.workers.diagram.network'}}
+      @color='highlight'
     >
-      <D.Item @icon='user'>
-        {{t 'resources.target.workers.diagram.client'}}
-      </D.Item>
       <D.Item @icon='settings'>
-        {{t 'resources.target.workers.diagram.ingress-worker'}}
+        {{t 'resources.target.workers.diagram.egress-worker'}}
       </D.Item>
       <D.Item @icon='server'>
         {{t 'resources.target.workers.diagram.host'}}
       </D.Item>
-    </OrderedSeriesDiagram>
-    <div class='target-worker-filter-explainer'>
-      <Hds::Form::HelperText>
-        {{t 'resources.target.workers.filter-explainer.ingress-worker'}}
-      </Hds::Form::HelperText>
-    </div>
-  {{else if @egressFilter}}
-    <OrderedSeriesDiagram
-      data-test-dual-filter-hcp-egress-on-ingress-off
-      as |D|
-    >
-      <D.Item @icon='user'>
-        {{t 'resources.target.workers.diagram.client'}}
-      </D.Item>
-      <D.Item @icon='settings'>
-        {{#if (feature-flag 'target-worker-filters-v2-hcp')}}
-          {{t 'resources.target.workers.diagram.hcp-worker'}}
-        {{else}}
-          {{t 'resources.target.workers.diagram.frontline-worker'}}
-        {{/if}}
-      </D.Item>
-      <D.Group
-        @title={{t 'resources.target.workers.diagram.network'}}
-        @color='highlight'
-      >
-        <D.Item @icon='settings'>
-          {{t 'resources.target.workers.diagram.egress-worker'}}
-        </D.Item>
-        <D.Item @icon='server'>
-          {{t 'resources.target.workers.diagram.host'}}
-        </D.Item>
-      </D.Group>
-    </OrderedSeriesDiagram>
-    <div class='target-worker-filter-explainer'>
-      <Hds::Form::HelperText>
-        {{t 'resources.target.workers.filter-explainer.hcp-dual-egress-on'}}
-      </Hds::Form::HelperText>
-    </div>
-  {{else}}
-    <OrderedSeriesDiagram
-      data-test-dual-filter-hcp-egress-off-ingress-off
-      as |D|
+    </D.Group>
+  </OrderedSeriesDiagram>
+  <div class='target-worker-filter-explainer'>
```

Comment:
Can these class names be namespaces in a way that is consistent with the component?  e.g. `worker-diagram-explainer`